mkosi.md

```diff
@@ -1316,6 +1316,51 @@ EOF
 # systemd-nspawn -bi image.raw
 ```
 
+## Different ways to boot with `qemu`
+
+The easiest way to boot a virtual machine is to build an image with the
+required components and let `mkosi` call `qemu` with all the right options:
+```console
+$ mkosi -d fedora \
+    --autologin \
+    -p systemd-udev,systemd-boot,NetworkManager,dracut,kernel-modules-core,kernel-core \
```

Comment:
Is `kernel-modules-core` required here? I would think `kernel-core` would have a dependency on `kernel-modules-core`? I'd also drop `NetworkManager` since it isn't required to boot a virtual machine